packages/model/src/research-output.ts

```diff
@@ -218,6 +218,7 @@ export type ResearchOutputDataObject = ResearchOutputCoreObject & {
   subtype?: string;
   teams: Pick<TeamResponse, 'id' | 'displayName'>[];
   workingGroups: Pick<WorkingGroupResponse, 'id' | 'title'>[];
+  published?: boolean;
```

Comment:
why optional? The research output is either published or not, don't see how it could come back as undefined?